staging/src/k8s.io/kube-gen/Godeps

```diff
@@ -239,6 +247,18 @@
 			"Rev": "f2499483f923065a842d38eb4c7f1927e6fc6e6d"
 		},
 		{
+			"ImportPath": "golang.org/x/sys/unix",
```

Comment:
quite a number of dependencies. Can you measure the binary (e.g. hyperkube) size before and after the PR?